index.html

```diff
@@ -12,5 +13,74 @@
+  <div class="container">
+    <header>
+      <nav class="style-nav">
+          <a href="services.html">Services</a>
+          <a href="#">Product</a>
+          <a href="#">Vision</a>
+          <a href="#">Features</a>
+          <a href="#">About</a>
+          <a href="#">Contact</a>
+      </nav>
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </header>
+
+    <section class="top-content">
+      <div class="welcome">
+        <span class="part-1"><p>Innovation</p></span>
+        <span class="part-2"><p>On</p></span>
+        <span class="part-3"><p>Demand</p></span>
+        <button>Get Started</button>
+      </div>
+      <div class="top-content-image">
+        <img src="img/header-img.png" alt="Image of a code snippet.">
+      </div>
+    </section><br>
+    <hr>
+
+    <section class="mid-content">
+      <div class="mid-content-wrapper">
+        <div class="features">
+          <h2>Features</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+        <div class="about">
+          <h2>About</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+        </div>
+      </div>
+
+      <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    </section>
+
+    <section class="bottom-content">
+      <div class="services">
+        <h2>Services</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+      <div class="product">
+        <h2>Product</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+      <div class="vision">
+        <h2>Vision</h2>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+    </section>
+    <hr><br>
+    <footer>
+      <address>
+        <h2>Contact</h2>
+        <p>123 Way 456 Street</p>
+        <p>Somewhere, USA</p><br>
+        <p>1 (888) 888-8888</p><br>
+        <p>sales@greatidea.io</p><br>
+      </address><br><br>
+      <p class="copyright">Copyright Great Idea! 2018</p>
+
+      </body>
+    </footer>
+      </div>
 </body>
```

Comment:
so it appear you have a tag wrapping a single tag, this isn't really necessary. I would ditch the div with the container class and give the body tag a class of container